cfgov/unprocessed/js/organisms/FilterableListControls.js

```diff
@@ -66,11 +66,17 @@ function FilterableListControls( element ) {
     if ( _dom.classList.contains( 'o-filterable-list-controls' ) ) {
       multiSelects.forEach( function( multiSelect ) {
         multiSelect.addEventListener( 'expandBegin', function refresh() {
-          window.setTimeout( _expandable.transition.expand, 250 );
+          window.setTimeout(
+            _expandable.transition.expand.bind( _expandable.transition ),
+            300
```

Comment:
Why the change from 250 to 300?

Reply:
Also, I'm forgetting why we need the delay at all? During testing, the delay in resize is kind of distracting.

Reply:
Because we have to wait for the transition on the multiselect itself to complete to know what the new desired height is. I agree it's distracting, but I'm not sure what else we can do.

I _think_ I lengthened the delay for a similar reason? Had to be sure the multiselect was done animating. But it's possible it was conflated with other work I discarded. I'll try going back to 250.

Reply:
Aww right. We'll need to have the multiselect dispatch its own event that the transition is complete. You might add a `TODO` for that since we both agree this could be better. But better than the live bug!

Reply:
Reverted this to 250 and rebased on master.